RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.cc

```diff
@@ -0,0 +1,115 @@
+#include "RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.h"
```

Comment:
This could be just
```suggestion
#include "OnlineBeamSpotESProducer.h"
```
or, event better, include the contents of the header in this file.